remoteit/desktop
# [DESK-280][DESK-290][DESK-434] Disable auto update switch

### Description


Added auto update preference
Added saved preferences
Refactored open at login to use preferences
Refactored frontend controller import to emit
Minor UI style tweaks
Check for update on switch toggle

### Checklist





### Test plan



1. Sign in
2. Go to settings page
3. See and toggle the auto-update switch
4. This will be hard to test without an update request
5. You could look in the log found in ~/.remoteit.log/remoteit.combined.log and see messages when you click the toggle.

### Ticket



<https://remoteit.atlassian.net/browse/DESK-280>
<https://remoteit.atlassian.net/browse/DESK-290>
<https://remoteit.atlassian.net/browse/DESK-434>
